src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetection.swift

```diff
@@ -145,21 +146,38 @@ final class ExposureDetection {
 	func start(completion: @escaping Completion) {
 		self.completion = completion
 
+		#if EUROPEMODE
```

Comment:
you might want to document `EUROPEMODE` in https://github.com/corona-warn-app/cwa-app-ios/blob/develop/COMPILATION_CONDITIONS.md

Reply:
Thanks! Added!